roles/openldap/defaults/main.yml

```diff
@@ -0,0 +1,52 @@
+keystone:
+  ldap:
+    enabled: False 
+
+    # ldap server settings
+    url: ldap://0.0.0.0:389
```

Comment:
If this is being used/referenced by Keystone, this should not be in openldap defaults...it should be a default for keystone and/or a related role.  That said, I still don't think we want LDAP listening on all interfaces.  LDAP should be listening on a 1918 address that is fronted by haproxy.